R/control.ergm.R

```diff
@@ -542,12 +549,13 @@ control.ergm<-function(drop=TRUE,
 
                        MCMLE.min.depfac=2,
                        MCMLE.sampsize.boost.pow=0.5,
+                       MCMLE.varweight=0.5,
 
                        MCMLE.MCMC.precision=if(startsWith("confidence", MCMLE.termination[1])) 0.1 else 0.005,
                        MCMLE.MCMC.max.ESS.frac=0.1,
                        MCMLE.metric=c("lognormal", "logtaylor",
                          "Median.Likelihood",
-                         "EF.Likelihood", "naive"),
+                         "EF.Likelihood", "Kpenalty", "naive"),
```

Comment:
Point of clarification: is Kpenalty an alternative way to estimate the likelihood ratio, or is it estimating the penalised likelihood?

Reply:
Kpenalty estimates the penalized likelihood. Explicitly, is computed the log-likelihood ratio (using `lognormal`) and then the kurtosis estimates to add to the log-likelihood ratio as the penalty. Otherwise, it is close to the standard `lognormal` version. The gradient is computed (mostly) exactly.